bibcd.yml

```diff
@@ -3,8 +3,8 @@ context: mlops
 triggering: master-and-prs
 
 defaultNodes:
-  cdBuild: m-asdf
-  prBuild: m-asdf
+  cdBuild: l-asdf
+  prBuild: l-asdf
```

Comment:
Why is this change needed?

Reply:
prBuild often seems to get killed when installing python dependencies, often with the error message: 
```
Connection pool is full, discarding connection: pypi.org. Connection pool size: 10
```
Suspect it's running out of memory, hence the increase in node size

Reply:
are we certain this solves the problem though? Also, this seems very unrelated to this PR and isn't even mentioned in the PR description.

Reply:
It appears to solve the problem, prBuild was flaky before I made that change and now it's not. I'm not sure how to prove it otherwise. I'll update the PR description though